<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>200</height>
   </rect>
  </property>
  
  <property name="windowTitle">
   <string>Wave Tank</string>
  </property>
  
  
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>129</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stroke length</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stroke period</string>
    </property>
   </widget>

   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of strokes</string>
    </property>
   </widget>


   <widget class="QDoubleSpinBox" name="strokeLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   
   <widget class="QDoubleSpinBox" name="strokePeriodSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
   </widget>

   <widget class="QSpinBox" name="numberOfStrokesSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  
  
  
  
  
  <widget class="QPushButton" name="squareWaveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Square wave</string>
   </property>
  </widget>

  <widget class="QPushButton" name="cosineWaveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cosine wave</string>
   </property>
  </widget>

  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>

  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>


  
 </widget>
 <resources/>
 <connections/>
</ui>

